documentation/variables.asciidoc

```diff
@@ -32,4 +32,5 @@ Please note that we are trying to minimize any potential side-effect from `devon
 |`«TOOL»_VERSION`|`-`|The version of the tool `«TOOL»` to install and use (e.g. `ECLIPSE_VERSION` or `MAVEN_VERSION`).
 |`«TOOL»_BUILD_OPTS`|e.g.`clean install`|The arguments provided to the build-tool `«TOOL»` in order to run a build.
 |`«TOOL»_RELEASE_OPTS`|e.g.`clean deploy -Dchangelist= -Pdeploy`|The arguments provided to the build-tool `«TOOL»` in order to perform a release build.
+|`DEVON_IDE_TRACE`||If value is not an empty string, the `devonfw-ide` scripts will trace each script line executed. For bash two lines output: before and again after expansion.
```

Comment:
Excellent, that you also added the documentation.
However, the way you implemented the feature this is not a regular variable like other variables in `devonfw-ide` as it can only be set upfront via `export DEVON_IDE_TRACE=true` while setting it in `devon.properties` as `DEVON_IDE_TRACE=true` will IMHO not work.
If you see this now and can quickly provide an update, that would be highly appreciated. Otherwise I might need to redo this PR to take over and add small changes.

Reply:
You are right - I just focussed on the table header. Using it with `export` inside `devon.properties` - but of course, this starts tracing only after variable import - which might be already too late.

I still think, that the variables documentation is a good place to mention this variable - but another new documentation file that describes the  tracing would of course also help. I'll provide an update soon.

Reply:
We are in the process of releasing devonfw 2020.08 roughly next week. If you can not provide the update soon enough, I will take over. Maybe I will just merge and then quickly edit the docs immediately after via browser to keep it simple.

Reply:
please do so